ajikurian-dublin.md

```diff
@@ -0,0 +1,11 @@
+### Dublin, California
```

Comment:
@ajikurian, I was stayed near Tracy for couple of months, and I like the train commute.